languages/translations/fr.json

```diff
@@ -118,367 +118,367 @@
         "continue": {
             "default": {
                 "default": "Tourner {modifier}",
-                "name": "Tourner {modifier} pour rester {way_name}",
-                "destination": "Tourner {modifier} en direction de {destination}",
-                "exit": "Tourner {modifier} sur {way_name}"
+                "name": "Tourner {modifier} pour rester {way_name:article}",
```

Comment:
The correct translation would be `Tourner {modifier} pour rester sur {way_name:article}`.